### PR TITLE
Centos6 support

### PR DESCRIPTION
**Pull request by tdooner**
@sds @z3usy This is still a WIP but here's the beginning of CentOS6 support for synapse.

### CODE REVIEW

File: test/Dockerfile-CentOS7 [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Specifies a Docker image suitable for testing cookbooks on CentOS 7.
+#
+# This handles a number of idiosyncracies with Systemd so it can be run as the
+# root process of the container, making it behave like a normal VM but without
+# the overhead.
+
+FROM centos:centos7
```
Comment (sds) [inline test/Dockerfile-CentOS7]:
```
FROM centos:centos7.0.1406
```
!!!!!!!!!!

**Comment by sds** [comment]
Looks good to me. Anything in particular you wanted us to look closer at?

File: test/Dockerfile-CentOS7 (hunk 1 above)
Comment (tdooner) [inline test/Dockerfile-CentOS7]: Done. Argh.

**Comment by tdooner** [comment]
@sds @z3usy Heads up about the two commits I just pushed here. These tests pass locally, but I don't have them integrated in any automated way. Nonetheless, I think this is ready to merge on my end, so if you get a chance could you look at the stuff I just added in those commits?

**Comment by tdooner** [comment]
All right, time to go.

**Pull request merged**